File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/VpsSnapshot.scala

package org.openapitools.client.model


case class VpsSnapshot (
    /* Snapshot name. */
    _name: Option[String],
    /* Disk space used by this snapshot in bytes. */
    _used: Option[Integer],
    /* Unix timestamp of when the snapshot was created. */
    _date: Option[Integer]
)
object VpsSnapshot {
    def toStringBody(var_name: Object, var_used: Object, var_date: Object) =
        s"""
        | {
        | "name":$var_name,"used":$var_used,"date":$var_date
        | }
        """.stripMargin
}

File: OPENAPI-REST-API/openapi-client/zapier/models/VpsSnapshot.js
const utils = require('../utils/utils');

module.exports = {
    fields: (prefix = '', isInput = true, isArrayChild = false) => {
        const {keyPrefix, labelPrefix} = utils.buildKeyAndLabel(prefix, isInput, isArrayChild)
        return [
            {
                key: `${keyPrefix}name`,
                label: `Snapshot name. - [${labelPrefix}name]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}used`,
                label: `Disk space used by this snapshot in bytes. - [${labelPrefix}used]`,
                type: 'integer',
            },
            {
                key: `${keyPrefix}date`,
                label: `Unix timestamp of when the snapshot was created. - [${labelPrefix}date]`,
                type: 'integer',
            },
        ]
    },
    mapping: (bundle, prefix = '') => {
        const {keyPrefix} = utils.buildKeyAndLabel(prefix)
        return {
            'name': bundle.inputData?.[`${keyPrefix}name`],
            'used': bundle.inputData?.[`${keyPrefix}used`],
            'date': bundle.inputData?.[`${keyPrefix}date`],
        }
    },
}
